chroma-manager/chroma-manager.spec

```diff
@@ -46,7 +46,6 @@ Requires: Django >= 1.4
 Requires: Django-south >= 0.7.4
 Requires: django-tastypie = 0.9.11
 Requires: django-picklefield
-Requires: chroma-manager-libs = %{version}-%{release}
```

Comment:
because it relies on chroma-manager-cli which in turn relies on the libs package